README.md

```diff
@@ -16,6 +16,14 @@ I recommend creating a separate package inside your project to serve as the cont
 
 For an example of how to use the resulting package, check out `example/example.go`. You can also see the API it generates at [godoc.org](https://godoc.org/github.com/bouk/staticfiles/files).
 
+## Installation
+
+Install with
+
+```
+go get github.com/bouk/staticfiles
```

Comment:
I strongly recommend including the `-u` flag here by default. It tends to cause fewer problems for users.

Those who know what they're doing and don't want to update to latest available version can run it without `-u` flag.